gslib/tests/test_ls.py

```diff
@@ -1091,3 +1091,14 @@ def test_list_public_access_prevention(self):
     self.RunGsUtil(['pap', 'set', 'enforced', suri(bucket_uri)])
     stdout = self.RunGsUtil(['ls', '-Lb', suri(bucket_uri)], return_stdout=True)
     self.assertRegex(stdout, r'Public access prevention:\t*enforced')
+
+  @SkipForXML('Autoclass is not supported for the XML API.')
+  @SkipForS3('Autoclass is not supported for S3 buckets.')
+  def test_list_autoclass(self):
+    bucket_uri = self.CreateBucket()
+    stdout = self.RunGsUtil(['ls', '-Lb', suri(bucket_uri)], return_stdout=True)
+    self.assertNotIn('Autoclass', stdout)
+    # Enforce public access prevention.
```

Comment:
I think this comment should be updated or removed.

Reply:
Good catch again!